wemake_python_styleguide/logic/tree/variables.py

```diff
@@ -16,12 +16,22 @@ def is_valid_block_variable_definition(node: _VarDefinition) -> bool:
     """Is used to check either block variables are correctly defined."""
     if isinstance(node, ast.Tuple):
         return all(
-            _is_valid_single(var_definition)
+            is_valid_block_variable_definition(var_definition)
             for var_definition in node.elts
         )
     return _is_valid_single(node)
 
 
+def is_valid_unpacking_target(target: ast.expr) -> bool:
+    """Checks if unpacking target is correct."""
+    if isinstance(target, ast.Tuple):
+        return all(
+            _is_valid_single(element) is not None
```

Comment:
`_is_valid_single` returns `bool` so why the result is compared to `None`?
Shouldn't it be just `_is_valid_single(element)`?